deploy/clusterwide/role.yaml

```diff
@@ -6,10 +6,10 @@ rules:
 - apiGroups:
```

Comment:
The diff on this file is a bit noisy, not your fault. Can you summarize what changed? I see we completely removed permissions on secrets but is there a specific reason we keep secrets in with `verbs: []`?

Reply:
Removed  create for pods, Removed everything from secret